etherman/etherman.go

```diff
@@ -164,7 +158,7 @@ func NewClient(cfg Config, polygonBridgeAddr, polygonZkEVMGlobalExitRootAddress,
 		OldPolygonBridge:           oldpolygonBridge,
 		PolygonZkEVMGlobalExitRoot: polygonZkEVMGlobalExitRoot,
 		PolygonRollupManager:       polygonRollupManager,
-		RollupID:                   rollupID,
+		rollupID:                   0,
```

Comment:
rollup ID used to return the other ID (not the one the etherman is connected to). This was confusing as f and I've decided to reverse the logic to return the rollup ID from the netwrok were Etherman is connected.

And for L1 this should always be 0